base/src/main/java/com/mindera/skeletoid/routing/OpenUriCommand.kt

```diff
@@ -1,19 +1,26 @@
 package com.mindera.skeletoid.routing
 
+import android.content.ActivityNotFoundException
 import android.content.Context
 import android.content.Intent
 import android.net.Uri
+import com.mindera.skeletoid.logs.LOG
 
 class OpenUriCommand(private val context: Context, private val deepLink: String) : IRouteCommand {
 
     companion object {
+        private const val LOG_TAG = "OpenUriCommand"
         const val IS_INTERNAL_APP_NAVIGATION_EXTRA = "IS_INTERNAL_APP_NAVIGATION_EXTRA"
     }
 
     override fun navigate() {
         val intent = Intent(Intent.ACTION_VIEW)
         intent.data = Uri.parse(deepLink)
         intent.putExtra(IS_INTERNAL_APP_NAVIGATION_EXTRA, true)
-        context.startActivity(intent)
+        try {
```

Comment:
nitpicking: why not use `runCatching` ?

Reply:
What are the advantages of that?

Reply:
just a more functional way of doing things in a Kotlin fashion. there are some cases that this format still has its advantages (when we want to catch just some exceptions), but for this one doesn't seem like it